.gitignore

```diff
@@ -1,3 +1,6 @@
+# yarn
+yarn.lock
```

Comment:
I think usually the `yarn.lock` file is committed

Reply:
Yep, fair point - this project uses `npm` apparently, and I use `yarn` out of habit. Didn't mean to check this in. Good catch!